### PR TITLE
COMMISS.ENH: Improve BbB classes.

### PR DESCRIPTION
**Pull request by fernandohds564**
tested with real data acquisition and processing.

### CODE REVIEW

**Review by anacso17 — APPROVED** [review]

File: apsuite/commisslib/measure_bbb_data.py [hunk 1]
```diff
@@ -525,6 +627,237 @@ def waterfall_plot(axis, xs, zs, data):
...
+            modes_stg = ', '.join([f'{m:03d}' for m in modes])
+        stg += stmp(
+            'modes_to_measure', f'[{modes_stg:s}] (size = {len(modes):d})')
+        return stg
+
+
+class DriveDampHParams(DriveDampLParams):
+    """."""
+
+    DAMPING_RATE = 1/16.9e-3  # Hz
```
Comment (xresende) [inline apsuite/commisslib/measure_bbb_data.py:667]: `[Hz]`, to keep the standard

File: apsuite/commisslib/measure_bbb_data.py [hunk 2]
```diff
@@ -525,6 +627,237 @@ def waterfall_plot(axis, xs, zs, data):
...
+    CALIBRATION_FACTOR = 1000  # Counts/mA/um
```
Comment (xresende) [inline apsuite/commisslib/measure_bbb_data.py:668]: [Countes/mA/um]

File: apsuite/commisslib/measure_bbb_data.py [hunk 3]
```diff
@@ -525,6 +627,237 @@ def waterfall_plot(axis, xs, zs, data):
...
+
+
+class DriveDampVParams(DriveDampLParams):
+    """."""
+
+    DAMPING_RATE = 1/22.0e-3  # Hz
```
Comment (xresende) [inline apsuite/commisslib/measure_bbb_data.py:674]: same here

File: apsuite/commisslib/measure_bbb_data.py [hunk 4]
```diff
@@ -525,6 +627,237 @@ def waterfall_plot(axis, xs, zs, data):
...
+        tim = infos['time']
+
+        if not self.data:
+            self.data = dict(infos=[], modes_data=[], modes_measured=[])
+        for mode in modes_to_measure:
+            drive.mask = _np.cos(2*_np.pi*bunches*mode/harm_num) > 0
+            _time.sleep(self.params.wait_acquisition)
+            bbb.sram.cmd_data_dump()
+
+            _time.sleep(0.2)
```
Comment (xresende) [inline apsuite/commisslib/measure_bbb_data.py:839]: a reminder to fix this value or implement callback, as you already figured out.

**Review by murilobalves — APPROVED** [review]

**Review by anacso17 — APPROVED** [review]

**Review by xresende — APPROVED** [review]

**Pull request merged**